searchlib/src/vespa/searchlib/queryeval/executeinfo.h

```diff
@@ -0,0 +1,29 @@
+// Copyright 2019 Oath inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+#pragma once
+
+namespace search::queryeval {
+
+    /**
+     * Holds information about how query will be executed and how large part of corpus will pass through.
+     * @author baldersheim
+     */
+class ExecuteInfo {
+public:
+    ExecuteInfo() : ExecuteInfo(false, 1.0) { }
+    ExecuteInfo(bool strict, double hitRate_in)
```

Comment:
consider hiding this if the create function is preferred.

Reply:
Hidden